apps/condo/domains/miniapp/access/SendAppPushMessageService.js

```diff
@@ -0,0 +1,16 @@
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canSendAppPushMessage ({ authentication: { item: user } }) {
```

Comment:
You should check that the `user` and `args.user` has the same id! **Only the user can send himself a push**

Reply:
This one is a mutation for sending messages to users on behalf of a B2C minApp

As far as I understand everything in our system is executed on behalf of some user
And I also assumed that user is the `user` the mutation is executed on behalf of, and `args.user` is the user who receives the push. So why should the user send the push to himself?